braille-printer-app/brf-printer-app.c

```diff
@@ -0,0 +1,1071 @@
+// Include necessary headers
+#define _GNU_SOURCE
+#include <strings.h>
+#include <cupsfilters/log.h>
+#include <cupsfilters/filter.h>
+#include <ppd/ppd-filter.h>
+#include <limits.h>
+#include <pappl/pappl.h>
+#include <liblouisutdml/liblouisutdml.h>
+#include <cups/backend.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <pwd.h>
+#include <string.h>
+#include <cups/ipp.h>
+#include <magic.h>
+
+#include "brf-printer.h"
+
+// Include necessary headers...
+
+#define brf_TESTPAGE_MIMETYPE "application/vnd.cups-brf"
```

Comment:
Remove the  `_TESTPAGE` part in the macro name

Reply:
And move the macro definition to the `.h` file, instead of having it defined in several `.c` files